python-intermediate-1.ipynb

```diff
@@ -224,8 +224,8 @@
     "Create a list ```new_list2``` which contains those numbers from the list ```numbers```, which, after being timed by 5, is smaller than 35. \n",
```

Comment:
<p>This looks like a slightly different table.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ithaka/tdm-notebooks/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>